guides/testing/index.md

```diff
@@ -1,78 +1,78 @@
-Ember gives you **the power to write tests and be productive from day one**. You can be confident that your app will be correct today and years from now. A question remains: _How_ should you write tests?
+Ember vous donne **la puissance d'écrire des tests et d'être productif dès le premier jour**. Vous pouvez être sûr que votre application fonctionnera correctement aujourd'hui et dans des années. Une question demeure : Comment devriez-vous écrire des tests ?
 
-Since tests are a core part of the Ember framework and your development cycle, we will dedicate several sections to learning how to write tests.
+Comme les tests sont un élément central du framework Ember et de votre cycle de développement, plusieurs sections sont consacrées à l'apprentissage de la rédaction de tests.
 
-In this section, we will cover why testing is important and how to run, debug and filter your tests.
+Dans cette section, nous aborderons les raisons pour lesquelles les tests sont importants et comment exécuter, déboguer et filtrer vos tests.
 
-## Why Do I Need Tests?
+## Pourqoi ai-je besoin de tests ?
 
-Writing tests is a necessary ingredient if you want to guarantee users and stakeholders that your app, whether small or large, will function as intended at any given time. The larger your app, the more costly and error-prone manual testing becomes.
+Écrire des tests est un ingrédient nécessaire si vous voulez garantir aux utilisateurs et aux parties prenantes que votre application, petite ou grande, fonctionnera comme prévu à tout moment. Plus votre application est volumineuse, plus les tests manuels deviennent coûteux et sujets aux erreurs.
 
-Writing tests is also a fun activity, a nice change of pace from delivering features daily, and a way to help you refactor code and improve as a developer. Tests can also serve as a living documentation — a key element in onboarding new developers.
+Écrire des tests est également une activité amusante, un bon changement de rythme par rapport à la livraison quotidienne de fonctionnalités. C'est aussi un moyen de vous aider à refactorer du code et à vous améliorer en tant que développeur. Enfin, Les tests peuvent vous servir de documentation vivante, un élément clé pour l'intégration de nouveaux développeurs.
```

Comment:
2 small proposals here:
- Turning the last statement into a new sentence is a very good idea 👍 In this case, you need to replace the previous coma with "et"
- "l'intégration" sounds a bit too technical, I think "intégrer à l'équipe" is a little better in this context. Also, try to avoid masculine terms for people as much as you can. We generally avoid "·" character when possible, but here I didn't find any better idea, it wasn't an easy one ^^ 

```suggestion
Écrire des tests est également une activité amusante et un bon changement de rythme par rapport à la livraison quotidienne de fonctionnalités. C'est aussi un moyen de vous aider à refactorer du code et à vous améliorer en tant que développeur. Enfin, les tests peuvent vous servir de documentation vivante, un élément clé pour intégrer de nouve·aux·lles devs à l'équipe.
```